src/Http/GraphRequest.php

```diff
@@ -466,7 +466,7 @@ public function upload($path, $client = null)
         try {
             if (file_exists($path) && is_readable($path)) {
                 $file = fopen($path, 'r');
-                $stream = \GuzzleHttp\Psr7\Utils::streamFor($file);
+                $stream = \GuzzleHttp\Psr7\stream_for($file);
```

Comment:
Is there any reason for reverting back to this? I can see the function has been marked deprecated since 1.x.

Reply:
Keeping the Utils methods would still be a breaking change.

`guzzlehttp\psr7` versions 1.0-1.7 do not support the Utils methods.

Sample scenario: a customer takes a direct dependency on `guzzlehttp\psr7` versions 1.0-1.7, our previous versions of the SDK (<=1.34.0) would have worked because they use the non-utils stream_for()

Reverting to the deprecated method would maintain backward compatibility in these scenarios